spec/features/cases/overturned_foi/mark_as_sent_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
```

Comment:
Sorry to say this Becca, but I think all these should really be written as controller tests, not feature tests.  They will run much more quickly, and I think we should reserve these headless browser tests for full journeys. What do you think?

Reply:
Sounds goooood!